apps/src/dance/lab2/ProgramExecutor.ts

```diff
@@ -3,7 +3,7 @@ import {SongMetadata} from '../types';
 import {commands as audioCommands} from '@cdo/apps/lib/util/audioApi';
 import * as danceMsg from '../locale';
 import {ASSET_BASE} from '../constants';
-import Lab2MetricsReporter from '@cdo/apps/lab2/Lab2MetricsReporter';
+import {LabMetricsReporter} from '@cdo/apps/lab2/Lab2MetricsReporter';
```

Comment:
should this be the default export (the instantiated instance) rather than the class?

Reply:
Ah no so I'm using the class export here so that we can pass in either the Lab2MetricsReporter or a different metrics reporter based on use case. This was originally intended to be only used in Lab2 Dance, but we now also use it to render the live preview in the AI modal in legacy Dance. So in legacy Dance AI, we can pass the DanceMetricsReporter, whereas in Lab2 Dance we can pass in the Lab2MetricsReporter.